README.md

```diff
@@ -86,11 +93,41 @@ If you are working in this repository, you just need to activate the environment
 
 ```sh
 ruff check .
-```
-```sh
 ruff format . --check
 ```
+
 Ruff auto-formatting is also a part of the GitHub workflow actions. Make sure that before you make a Pull Request, `ruff format . --check` runs in your local without any errors otherwise the workflow action will fail.
 
 Alternatively, if you are using VSCode as your IDE, we added the settings configuration file, `.vscode/settings.json`, such that it performs `ruff format` whenever you save progress in a file.
 
+### Debugging
+
+For interactive debugging of the tests, use `pytest` with the `--pdb` flag.
+The officially recommended IDE is VSCode..
+When debgugging in there, add the following snippet to the `launch.json` jobs:
```

Comment:
"We recommend using an IDE for debugging, e.g., VSCode. If using VSCode, you can add a snipped in `.vscode/launch.json` with the following content:
```json
...
```

where `${workspaceFolder}` is considered to be the root folder of this repository.

The settings configuration file `.vscode/settings.json` performs `ruff format` automatically whenever you save progress in a file.
"

Reply:
Please, note that the workspaceFolder, if added in a launch.json here is pointing to the workspace folder **of the repository** itself.

Reply:
> where ${workspaceFolder} is considered to be the root folder of this repository.

No, it refers to the NOMAD root.

Reply:
But which launch.json are you refering to? You can set up a python venv directly in this plugin, and work from it without the need of having a nomad local installation

Reply:
Haven't tested that case, can't provide a setup script there.

Reply:
but if this is a matter for the nomad general repo, I disagree on including these instructions here; they are confusing tho